dvtemplates/new_cdi_image.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -e
+
+ERROR_BASE_URL_UNREACHABLE=1
+ERROR_INVALID_IMAGE=2
+ERROR_IMAGE_URL_UNREACHABLE=3
+ERROR_IMAGE_EXISTS=4
+ERROR_LOCAL_REGISTRY=5
+ERROR_QUAY_REGISTRY=6
+ERROR_TESTS_FAIL=7
+ERROR_MISSING_OS_IMAGE=8
+
+error_message() {
+    EXIT_VAL=$1
+    OS="$2"
+    if [ $EXIT_VAL -eq $ERROR_BASE_URL_UNREACHABLE ]; then
+        echo -e "Unable to access data from the OS release URL"
+    elif [ $EXIT_VAL -eq $ERROR_INVALID_IMAGE ]; then
+        echo -e "Invalid Image Version"
+    elif [ $EXIT_VAL -eq $ERROR_IMAGE_URL_UNREACHABLE ]; then
+        echo -e "Unable to connect to the OS Image repository"
+    elif [ $EXIT_VAL -eq $ERROR_IMAGE_EXISTS ]; then
+        echo -e "Container Image for the Latest $OS version is already present. Exiting"
+	exit 0
+    elif [ $EXIT_VAL -eq $ERROR_LOCAL_REGISTRY ]; then
+        echo -e "Unable to push the image to the local Registry."
+    elif [ $EXIT_VAL -eq $ERROR_QUAY_REGISTRY ]; then
+        echo -e "Unable to push the image to the Quay Registry."
+    elif [ $EXIT_VAL -eq $ERROR_TESTS_FAIL ]; then
+        echo -e "End to End Tests fail."
+    elif [ $EXIT_VAL -eq $ERROR_MISSING_OS_IMAGE ]; then
+        echo -e "No OS Image is set. Exiting!"
+    fi
+    exit $EXIT_VAL
+}
+
+trap 'error_message $?' EXIT
+
+#Insecure registry port
+port=5000
+
+# Check if TARGET_OS is set
+if [ -z "$TARGET_OS" ]; then
+    error_message "$ERROR_MISSING_OS_IMAGE"
+fi
+
+FEDORA_OS="fedora"
+CENTOS="centos8"
+
+# Set the Image registry and identify the latest version of the OS
+if [[ "${TARGET_OS}" == "${CENTOS}" ]]; then
+    cd "${PWD}/centos8"
+    OS_REPO="quay.io/kubevirt/centos8-container-disk-images"
```

Comment:
Are the two repos of a different naming schema on purpose?

Reply:
Yea, I thought this name made more sense. I can check with Federico and update the Fedora repo in a follow-up PR as well.